extension/tamper.css

```diff
@@ -31,6 +31,7 @@ h1 {
 label {
   display: block;
   padding: 5px 0;
+  color: blue;
```

Comment:
As much as I tried, I had a hard time with devtools.panels.themeName (I'm on Version 58.0.3029.81). I do agree that this is worthy of a more elegant solution and I do intend to play with it a bit more, but I think the path of least resistance for a quick fix would be to just hardcode the bg color as you mentioned above. I went with grey / white. Here's a screenshot. Happy to make any updates you suggest.
![screen shot 2017-04-30 at 7 56 02 pm](https://cloud.githubusercontent.com/assets/16121304/25569174/ab317292-2ddf-11e7-8e43-8964afb9abb2.png)